### PR TITLE
Update updateVersion to use debian package

### PR DESCRIPTION
**Pull request by sebastienvas**
Another PR in test-infra will update the dependency update tools + release tool.
I am not updating the rest of auth nor pilot to avoid conflict. The update will take care of this once it is updated.

```release-note
none
```

### CODE REVIEW

**Comment by ldemailly** [comment]
can we make the URL be split into the part that doesn't change and then reuse the appropriate sha - for instance for istioctl url:
```
export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/$PILOT_TAG/artifacts/istioctl"
```
it's very hard to update by hand/error prone with the full URL

**Comment by ldemailly** [comment]
when the version check runs and fail, maybe if it did "git diff" it would help see what's up

**Comment by sebastienvas** [comment]
I attempted to resolve conflict from the UI. I added the diff.

File: install/updateVersion.sh [hunk 1]
```diff
@@ -204,5 +215,5 @@ fi
 
 if [[ ${CHECK_GIT_STATUS} == true ]]; then
   check_git_status \
-    || { echo "Need to update template and run install/updateVersion.sh"; exit 1; }
+    || { echo "Need to update template and run install/updateVersion.sh"; git diff; exit 1; }
```
Comment (ldemailly) [inline install/updateVersion.sh]: 👍

File: istio.VERSION [hunk 2]
```diff
@@ -7,4 +7,8 @@ export MIXER_TAG="4e107a2411e1d58e88fd9ae4d32a139751e87955"
 export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/istioctl"
 export PILOT_HUB="gcr.io/istio-testing"
 export PILOT_TAG="3ac357f84576f881d9f57ed76e4e14a052ac5333"
-export ISTIO_NAMESPACE=istio-system
+export ISTIO_NAMESPACE="istio-system"
+export AUTH_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/auth/b7b8c10bbb6ca471e9c8aee2b3889f0ccbf01c3c/artifacts/debs"
+export PILOT_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/debs"
+export PROXY_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/proxy/4a493c07e34e764e29013c794ad295f665d8eebc/artifacts/debs"
```
Comment (ldemailly) [inline istio.VERSION]: in a way this is the only one that doesn't have a _TAG already
I would add
```
export PROXY_TAG="xxxxxx"
export PROXY_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/proxy/${PROXY_TAG}/artifacts/debs"
```
and then it'll be easier to maintain and also is already compatible for auth and pilot debian with no change

Comment (sebastienvas) [reply istio.VERSION]: should we update them all such that url is a template ?

Comment (ldemailly) [reply istio.VERSION]: yes I think so (what I wrote above: https://github.com/istio/istio/pull/826#issuecomment-329915609 )

**Pull request merged**